### PR TITLE
Add scrapy examples to repository

### PR DESCRIPTION
**Pull request by yuvalofek**
Added examples to the repository:
* Added new readme section: 'Examples'
* Linked to quote-spider examples as seen in the docs
* Created Google Colab notebook to run scrapy 
* Added notebook to new examples folder and linked to readme

### CODE REVIEW

**Comment by Gallaecio** [comment]
I’m unsure about including a link to this external tutorial in the documentation. The documentation already has its own tutorial. If we were to link external tutorials, we should review most existing ones, as I figure there are quite a few, and choose the best ones. Still, having a built-in tutorial ensures we can keep it up to date with code changes, which we cannot do with external tutorials out of our control.

Moreover, I don’t seem to be able to access this notebook.

**Comment by yuvalofek** [comment]
> I’m unsure about including a link to this external tutorial in the documentation. The documentation already has its own tutorial. If we were to link external tutorials, we should review most existing ones, as I figure there are quite a few, and choose the best ones. Still, having a built-in tutorial ensures we can keep it up to date with code changes, which we cannot do with external tutorials out of our control.
> 
> Moreover, I don’t seem to be able to access this notebook.

My goal was to create a set of tutorial notebooks to match the current documentation, not as something new but rather as a supplement in the form of a python notebook that is easily accessible and easy to use. 
I wanted to add all notebooks to the repository directly so as to have them keep up to date with code changes and be within your control. 

As for the link, I was following: [https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb ](https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb) on a way to have the local notebook open directly in Google Colab. I didn't want to link it to my personal repository, so I modified it to work for the scrapy repo once the code was implemented. 

I thought it could be useful to have a notebook form of the tutorial, but if that is not the case do let me know.

**Comment by Gallaecio** [comment]
Oh, I did not realize what was going on here at all, sorry!

**Review by Gallaecio — COMMENTED** [review]
I like the idea as you described, but I think it may be best for the notebook to almost completely match the content of the tutorial, so that users can choose between the static tutorial or the notebook. What do you think?

File: examples/Scrapy_Tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,967 @@
...
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/yuvalofek/scrapy/blob/master/Scrapy_Tutorial.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
[the remark this answers is not in the document]
Comment (yuvalofek) [reply examples/Scrapy_Tutorial.ipynb:27]: You are right, the intension was to get the notebook in the repository, but I must not have changed the link

File: README.rst [hunk 2]
```diff
@@ -74,6 +74,14 @@ Documentation
...
 directory.
 
+Examples
+========
+There is an example Scrapy project named quotesbot, here: https://github.com/scrapy/quotesbot, that you can use to play and learn more about Scrapy.
+
+Other tutorials:
+
+* Introductory tutorial notebook: https://github.com/scrapy/scrapy/blob/master/examples/Scrapy_Tutorial.ipynb
+
```
[the remark this answers is not in the document]
Comment (yuvalofek) [reply README.rst:84]: I like that! I'll fix up the notebook so it follows the tutorial closely :)

[the remark this answers is not in the document]
Comment (yuvalofek) [reply README.rst:84]: Got a bit lost in changing up the pull request and ended up opening another one. (Add a jupyter notebook copy of the tutorial #5238)

**Pull request closed without merging**